### PR TITLE
Add option to not mule into personal, but shared stash (hardcore)

### PR DESCRIPTION
**Pull request by theksk23**
Hola,
a little something we hardcore players desire, not quite sure if we should put that behind a generic `character: hardcore` setting, or have something more specific like `character.muleToShared`? 

Thanks!

### CODE REVIEW

**Comment by theksk23** [comment]
Looking at that in this graphical manner: tabs or spaces?

**Comment by hectorgimenez** [comment]
Hola,

I think it's a good idea, I would say maybe better a `stashToShared` or something like this, since it will be more configurable? I assume `hardcore: true` in the future can contain also other stuff focused on hardcore playing, and maybe someone by any reason wants to stash on shared even if not playing hardcore, so I would prefer to keep things as specific as possible to cherrypick.

Gracias!

File: internal/action/stash.go [hunk 1]
```diff
@@ -117,6 +117,9 @@ func (b *Builder) stashGold(d data.Data) {
 
 func (b *Builder) stashInventory(d data.Data, forceStash bool) {
 	currentTab := 1
+  if b.CharacterCfg.Character.StashToShared {
```
Comment (13413j1j13j5315n13) [inline internal/action/stash.go:120]: Did you run go fmt on this file?

**Comment by theksk23** [comment]
me is a total Go noob,
Will look that up and run it, thanks for the hint!
(did disable Neovim autoformat because it was re-ordering imports, eh..)



13413j1j13j5315n13 ***@***.***> schrieb am Do. 11. Apr. 2024
um 11:32:

**Comment by theksk23** [comment]
that indeed fixed the formatting, thanks!

**Pull request merged**